command/agent/agent.go

```diff
@@ -15,11 +15,12 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/hashicorp/consul/agent/hcp"
 	"github.com/hashicorp/go-checkpoint"
 	"github.com/hashicorp/go-hclog"
 	mcli "github.com/mitchellh/cli"
 
+	"github.com/hashicorp/consul/agent/hcp"
```

Comment:
nit: I think this should get grouped with the other "hashicorp/consul" imports below.

Reply:
oops looks like I didn't have my commit hooks set up to fix that. Good catch.

Reply:
Kind of surprised we don't have a linter in CI for formatting